File: generated-sources/dart-jaguar/mojang-sessions/lib/model/player_texture_url.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'player_texture_url.jser.dart';

class PlayerTextureURL {
   /* The URL to the texture, must be in Mojang's domains. */
  @Alias('url')
  final String url;
  

  PlayerTextureURL(
    

{
    
     this.url = null 
    }
  );

  @override
  String toString() {
    return 'PlayerTextureURL[url=$url, ]';
  }
}

@GenSerializer()
class PlayerTextureURLSerializer extends Serializer<PlayerTextureURL> with _$PlayerTextureURLSerializer {

}

File: generated-sources/scalaz/mojang-sessions/src/main/scala/org/openapitools/client/api/PlayerTextureURL.scala
package org.openapitools.client.api

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import PlayerTextureURL._

case class PlayerTextureURL (
  /* The URL to the texture, must be in Mojang's domains. */
  url: String)

object PlayerTextureURL {
  import DateTimeCodecs._

  implicit val PlayerTextureURLCodecJson: CodecJson[PlayerTextureURL] = CodecJson.derive[PlayerTextureURL]
  implicit val PlayerTextureURLDecoder: EntityDecoder[PlayerTextureURL] = jsonOf[PlayerTextureURL]
  implicit val PlayerTextureURLEncoder: EntityEncoder[PlayerTextureURL] = jsonEncoderOf[PlayerTextureURL]
}
